### DISCUSSION TITLE
Questions about Total head BC and Seepage face BC

### DISCUSSION THREAD

**Discussion by Tu-Shiqin** [Anura3D/Anura3D_OpenSource#87] [Q&A] [answered]
Hi dear all,
I have some questions about the total head and seepage face boundary condition. Fig. 1 and Fig. 2 display the fix condition and range of MPs respectively.
![image](https://user-images.githubusercontent.com/125841317/226627425-ffac21a1-e579-4cbb-bd45-c685799de7aa.png)
Fig. 1
![image](https://user-images.githubusercontent.com/125841317/226627761-8cbdd525-3e16-41ce-a68e-7dbc09635d6a.png)
Fig. 2
The phreatic surface (include .HHBF file) and soil surface have been assign as shown in Fig.3 and Fig. 4.
![image](https://user-images.githubusercontent.com/125841317/226628517-389655bf-aaa2-4f3e-a67a-80ab1c759615.png)
Fig. 3
![image](https://user-images.githubusercontent.com/125841317/226628376-05d4a544-df95-475e-8564-2f5b283fc74e.png)
Fig. 4 
After initializing stress in the soil by K0-procedure, some specifed fixities both liquid and soil have been removed as shown in Fig.5.
![image](https://user-images.githubusercontent.com/125841317/226629958-96696494-4ead-4736-babf-6e48b1c4d466.png)
Fig. 5
And then the MPs were fixed by assigning 1 under the "$$FIX_SOLID_SKELETON" in .CPS, the contour of liquid pressure and hydraulic head after some time steps are shown in Fig. 6 and Fig. 7.
![image](https://user-images.githubusercontent.com/125841317/226631637-04d6c196-02e6-4595-b478-5df80ac375c3.png)
Fig. 6
![image](https://user-images.githubusercontent.com/125841317/226631519-688b5c80-5bee-4275-a4c9-8bb4b143f13d.png)
Fig. 7
It seems like that the hydraulic head near the free boundary decreased with a constant total head (.HHBF or phreatic surface was constant during calculation). But when the simulation continued with the unconstrained soil skeleton ("$$FIX_SOLID_SKELETON" is 0), the liquid pressure and hydraulic head near the free boundary increased to initial valus rapidly, as shown in Fig. 8 and Fig. 9. And what's even more surprising is that the MPs didn't collapse without fixities at the free boundary.
![image](https://user-images.githubusercontent.com/125841317/226633717-ea65c066-b4ae-4895-aa06-92e346719763.png)
Fig. 8
![image](https://user-images.githubusercontent.com/125841317/226633631-bd17e262-5d31-49dc-b4c5-e9f53443eebd.png)
Fig. 9
Then I found the explanation of Total head BC in ScientificManual_2022, as shown in Fig. 10. 
![image](https://user-images.githubusercontent.com/125841317/226635316-60222df9-cde5-402e-b308-d354af92d8f4.png)
![image](https://user-images.githubusercontent.com/125841317/226635480-974f04d7-99b1-4dff-91fd-496b4d662479.png)
Fig. 10

Questions:
1. Is the phreatic surface and .HHBF file corresponding to the Total head BC? When the phreatic surface or .HHBF file will determine the total head boundary condition in whole simulation?
2. Can I assign two or more discontinuesd phreatic surfaces in one model?
3. As explained in SceintificManual (Fig. 10), the liquid pressure in MP is solved by Eq. (5.5). Is that mean the Eq. (5.5) determines the liquid pressure in whole calculation process? But I want to know why the hydraulic head and liquid pressure varied rapidly after skeleton displaced slightly (in one time step with 0.01s). 
4. which will be selected as seepage face? Can I understand that the soil surface definied in Gid is evaporated boundary and the boundary nodes is seepage boundary. But when a free boundary is defined as shown in Fig. 5, it will be checked by "seepage face"?

If you need a further description of the problems, please leave comments below. I would be grateful if you could provide any suggestions. Thank you for your any help. 

Best 
Tu

**Comment by FrancescaCeccato** [comment, marked as answer]
Dear Tu-Shiqin,
I try to answer:
1. Phreatic surphace is only used to calculate the water pressure at the first time step. HBBF provide a total head for the entire calculation preocess (when it is applied) and it is a BOUNDARY CONDITION, meaning that it is applied at a specific surface.
2. no, we never tried. How can this happen in reality?
3. eq. 5.5. is only used to calulcate the pressure at the boundary where this hydraulic boundary condition is applied and not everywhere in the model. In you case pressure is 0 at the right of the hole becasue you are removing water fixity, which means that pressure becomes 0 at that boundary unless otherwise specified.
4.  seepage face, total head and infiltration boundary are hydraulic boundary condition and must be applied as described in the tutorial manual. I don't see any specificato for them in your model.

I hope this helps.
